save-plugins/fix-and-warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugins/fixandwarn/FixAndWarnPlugin.kt

```diff
@@ -88,7 +101,7 @@ class FixAndWarnPlugin(
         // TODO: then warn plugin should look at the fix plugin output for actual warnings, and not execute command one more time.
         // TODO: However it's required changes in warn plugin logic (it's should be able to compare expected and actual warnings from different places),
         // TODO: this probably could be obtained after https://github.com/cqfn/save/issues/164,
-        val warnTestResults = warnPlugin.handleFiles(expectedFiles.map { listOf(it) })
+        val warnTestResults = warnPlugin.handleFiles(expectedFilesWithoutFail.map { listOf(it) })
         return fixTestResults.asSequence() + warnTestResults
```

Comment:
If I correctly remember, we shouldn't return all results. We need to return only those files, on which fix plugin fails, and warn results on which fix plugin pass